homeassistant/components/glances/config_flow.py

```diff
@@ -72,11 +78,10 @@ async def async_step_user(
                 )
             except CannotConnect:
                 errors["base"] = "cannot_connect"
+            except AuthorizationError:
+                errors[CONF_USERNAME] = "invalid_auth"
+                errors[CONF_PASSWORD] = "invalid_auth"
```

Comment:
This means both fields directly show an error, I think just putting the error in "base" is enough

Reply:
This is to show that the error is coming from the username or password. Not the other fields.

Reply:
Right, if we would show "Invalid authentication" (which is the `invalid_auth` translation), would there be any other option people would look?

Reply:
Updated the errors as suggested.